grouper/fe/templates/permission-request-update.html

```diff
@@ -30,8 +30,12 @@
                 </tr>
                 <tr>
                     <td colspan="5">
-                        <span style="padding-left: 10px;">
-                            <strong>Reason:</strong> <em>{{comment.comment}}</em>
+                        <dl class="dl-horizontal" style="margin-bottom: 0px;">
+                            <dt>Permission:<dt>
+                            <dd><a href="/permissions/{{ request.permission.name }}">
+                                {{ request.permission.name }}</a></dd>
```

Comment:
We don't have a linter for our templates, but I think ideal style is to put closing tags on a separate line from content if the opening tags were.